screen2.0/src/common/components/MainResultsFilters.tsx

```diff
@@ -70,146 +70,38 @@ export default function MainResultsFilters(props: { mainQueryParams: MainQueryPa
   const [PLS, setPLS] = useState<boolean>(props.mainQueryParams.PLS)
   const [TF, setTF] = useState<boolean>(props.mainQueryParams.TF)
 
-  const router = useRouter()
-
-  //IMPORTANT: This will wipe the current cCRE when Nishi puts it in. Need to talk to Nishi about deciding when/how to display the cCRE details
-  /**
-   *
-   * @param newBiosample optional, use if setting Biosample State and then immediately triggering router before re-render when the new state is accessible
-   * @returns A URL configured with filter information
-   */
-  function constructURL(newBiosample?: { selected: boolean; biosample: string; tissue: string; summaryName: string }) {
-    //Assembly, Chromosome, Start, End
-    const urlBasics = `search?assembly=${props.mainQueryParams.assembly}&chromosome=${props.mainQueryParams.chromosome}&start=${props.mainQueryParams.start}&end=${props.mainQueryParams.end}`
-
-    //Can probably get biosample down to one string, and extract other info when parsing byCellType
-    const biosampleFilters = `&Tissue=${outputT_or_F(Tissue)}&PrimaryCell=${outputT_or_F(PrimaryCell)}&InVitro=${outputT_or_F(
-      InVitro
-    )}&Organoid=${outputT_or_F(Organoid)}&CellLine=${outputT_or_F(CellLine)}${
-      (Biosample.selected && !newBiosample) || (newBiosample && newBiosample.selected)
-        ? "&Biosample=" +
-          (newBiosample ? newBiosample.biosample : Biosample.biosample) +
-          "&BiosampleTissue=" +
-          (newBiosample ? newBiosample.tissue : Biosample.tissue) +
-          "&BiosampleSummary=" +
-          (newBiosample ? newBiosample.summaryName : Biosample.summaryName)
-        : ""
-    }`
-
-    const chromatinFilters = `&dnase_s=${DNaseStart}&dnase_e=${DNaseEnd}&h3k4me3_s=${H3K4me3Start}&h3k4me3_e=${H3K4me3End}&h3k27ac_s=${H3K27acStart}&h3k27ac_e=${H3K27acEnd}&ctcf_s=${CTCFStart}&ctcf_e=${CTCFEnd}`
-
-    const classificationFilters = `&CA=${outputT_or_F(CA)}&CA_CTCF=${outputT_or_F(CA_CTCF)}&CA_H3K4me3=${outputT_or_F(
-      CA_H3K4me3
-    )}&CA_TF=${outputT_or_F(CA_TF)}&dELS=${outputT_or_F(dELS)}&pELS=${outputT_or_F(pELS)}&PLS=${outputT_or_F(PLS)}&TF=${outputT_or_F(TF)}`
-
-    const url = `${urlBasics}${biosampleFilters}${chromatinFilters}${classificationFilters}`
-    return url
+  const urlParams = {
```

Comment:
This can be const urlParams = {
    Tissue,
    PrimaryCell,
    InVitro,
    Organoid,
    CellLine,
    Biosample: { selected: Biosample.selected, biosample: Biosample.biosample, tissue: Biosample.tissue, summaryName: Biosample.summaryName },
    DNaseStart,
    .......
  }

Reply:
Thanks, didn't know there was shorthand for this, will change